src/tutorials/0004-mutable-file-system/08.md

```diff
@@ -21,12 +21,24 @@ You can use `files.mv` to perform a number of different operations:
 // move a single file into a directory
 await ipfs.files.mv('/source-file.txt', '/destination-directory')
 
-// move multiple files into a directory
-await ipfs.files.mv('/source-file-1.txt', '/source-file-2.txt', '/destination-directory')
-
 // move a directory into another directory
 await ipfs.files.mv('/source-directory', '/destination-directory')
 
 // overwrite the contents of a destination file with the contents of a source file
 await ipfs.files.mv('/source-file.txt', '/destination-file.txt')
 ```
+
+To move multiple files into a directory, you can add multiple files before the `to` argument:
+
+```js
+// move multiple files into a directory
+await ipfs.files.mv('/source-file-1.txt', '/source-file-2.txt', '/source-file-3.txt', '/destination-directory')
+```
+
+You can add as many files as you want, but if you have an array of files, you can use the spread operator syntax instead:
```

Comment:
If you're starting with an array of source paths, you can use JavaScript's [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to expand its elements into the required format: